### PR TITLE
Hshami/version 0 14 4 rc0

### PR DESCRIPTION
**Pull request by mhshami01**
Update README and Version #

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
     "python.testing.unittestEnabled": true,
     "python.testing.pytestEnabled": false,
-    "python.pythonPath": "c:\\Work\\AzureIoT\\IoTEdge\\IoTEdgeHubDev\\py36venv\\Scripts\\python.exe",
```
Comment (marianan) [inline .vscode/settings.json:4]: >pythonPath [](start = 12, length = 10)

can you debug the tests w/o the pythonPath explicitly specified?

File: README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ The following table compares the requirements to run your solution on the IoT Ed
 [Windows](https://docs.docker.com/docker-for-windows/install/), [macOS](https://docs.docker.com/docker-for-mac/install/) or [Linux](https://docs.docker.com/install/linux/docker-ce/ubuntu/#install-docker-ce)
 
 2. Install [Docker Compose (1.20.0+)](https://docs.docker.com/compose/install/#install-compose) (***Linux only***. *Compose has already been included in Windows/macOS Docker CE installation*)
-3. Install [Python (2.7/3.5/3.6/3.7/3.8) and Pip](https://www.python.org/)
```
Comment (marianan) [inline README.md:26]: >2.7 [](start = 20, length = 3)

py 2.7 is still supported for iotedgehubdev, please don't remove it from the isntructions

**Review by marianan — CHANGES_REQUESTED** [review]
:clock1:

File: .vscode/settings.json (hunk 1 above)
Comment (mhshami01) [reply .vscode/settings.json:4]: Yes - this change is prompted by vs code complaining about that line. Either way, it was environment specific :-)

File: README.md (hunk 2 above)
Comment (mhshami01) [reply README.md:26]: Will do - thank you!

**Review by marianan — APPROVED** [review]
:shipit:

File: README.md (hunk 2 above)
Comment (smcd253) [reply README.md]: @mhshami01 same may be said of the tox.ini for the supported versions of python. I noticed you removed py27 from there.

**Pull request merged**